### PR TITLE
Add new configs for dev-latest.kubeflow.org via kfctl.sh

### PR DESCRIPTION
**Pull request by activatedgeek**
These are new configurations for the `kubeflow-latest` cluster.

/cc @richardsliu @jlewi

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/testing/193)

### CODE REVIEW

**Comment by activatedgeek** [comment]
@richardsliu Can you verify if the current `upgrade_ks` command is enough in `deploy_latest.sh`?

File: deployment/README.md [hunk 1]
```diff
@@ -1,23 +0,0 @@
-# Kubeflow deployment
```
Comment (richardsliu) [inline deployment/README.md]: Can you add the new instructions here?

File: dev-latest-kubeflow-org/deploy_latest.sh [hunk 2]
```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+# Required Variables
+export KUBEFLOW_REPO=${KUBEFLOW_REPO:-}
+export PROJECT=${PROJECT:-}
```
Comment (richardsliu) [inline dev-latest-kubeflow-org/deploy_latest.sh]: We should also parameterize the kubernetes namespace. Otherwise it defaults to "kubeflow".

Comment (activatedgeek) [reply dev-latest-kubeflow-org/deploy_latest.sh]: Thanks for pointing out!

**Comment by jlewi** [comment]
I have no additional comments.
This is good to go as soon as @richardsliu is satisfied.

**Comment by activatedgeek** [comment]
@jlewi Do we need to have a copy of the configs here? Because these can be directly generated via `kfctl` now without a laborious process and deployed on demand.

**Comment by jlewi** [comment]
Yes we need a copy of the configs.
Git should be the source of truth so there should be configs checked into source control matching what's deployed.

**Comment by jlewi** [comment]
Closing this PR because I don't think it is relevant anymore.

**Pull request closed without merging**